result/isislsp.yaml

```diff
@@ -0,0 +1,341 @@
+        is_type:
+          description: >-
+            Type of neighboring system.
+          type: integer
+          minimum: 1
+          maximum: 3
+        hostname:
+          description: >-
+            This container defines TLV 137.
+          type: string
+        is_reachability:
+          description: |-
+            This container describes list of ISIS neighbors and attributes.
+          $ref: '#/components/schemas/IsisLsp.IsReachability'
+        extended_is_reachability:
+          description: |-
+            This container describes list of ISIS neighbors and attributes.
+          $ref: '#/components/schemas/IsisLsp.ExtendedIsReachability'
+        ipv4_internal_reachability:
+          description: |-
+            This container defines list of IPv4 internal reachability information.
+            This is advertised when the origin-type is set 'internal' in route range configurations
+          $ref: '#/components/schemas/IsisLsp.Ipv4InternalReachability'
+        ipv4_external_reachability:
+          description: |-
+            This container defines list of IPv4 external reachability information.
+            This is advertised when the origin-type is set 'external' in route range configurations
+          $ref: '#/components/schemas/IsisLsp.Ipv4ExternalReachability'
+        extended_ipv4_reachability:
+          description: |-
+            This container defines list of IPv4 extended reachability information.
+            It is advertised when the 'wide metric' is enabled.
+          $ref: '#/components/schemas/IsisLsp.ExtendedIpv4Reachability'
+        ipv6_reachability:
+          description: |-
+            This container defines list of IPv6 reachability information.
+          $ref: '#/components/schemas/IsisLsp.Ipv6Reachability'
+
+    IsisLsp.Flags:
+      description: >-
+        LSP Type flags.
+      type: object
+      properties:
+        partition_repair:
+          description: |-
+            When set, the originator supports partition repair.
+          type: boolean
+        attached_error:
+          description: |-
+            When set, the originator is attached to another area using the referred metric.
+          type: boolean
+        attached_expense:
+          description: |-
+            When set, the originator is attached to another
+            area using the referred metric.
+          type: boolean
+        attached_delay:
+          description: |-
+            When set, the originator is attached to another
+            area using the referred metric.
+          type: boolean
+        attached_default:
+          description: |-
+            When set, the originator is attached to another
+            area using the referred metric.
+          type: boolean
+        overload:
+          description: |-
+            When set, the originator is overloaded, and must
+            be avoided in path calculation.
+          type: boolean
+
+    IsisLsp.IsReachability:
```

Comment:
IsReachability and externded is reachability describtions are same

Reply:
descriptions i mean

Reply:
updated with more details.